README.md

```diff
@@ -1,3 +1,49 @@
-## OrderlyWeb Deploy
+## OrderlyWeb (Deploy)
 
-Deploy scripts for orderly web
+[![Build Status](https://travis-ci.org/vimc/orderly-web-deploy.svg?branch=master)](https://travis-ci.org/vimc/orderly-web-deploy)
+[![codecov.io](https://codecov.io/github/vimc/orderly-web-deploy/coverage.svg?branch=master)](https://codecov.io/github/vimc/orderly-web-deploy?branch=master)
+
+This is the deploy scripts for OrderlyWeb.  They are the only part of the system that runs directly on metal.
+I am not good at python packaging so some documentation for help me.  These might not be the best ways to do things but they seem to work:
+
+## Installation
+
+From local sources
+
+```
+python3 setup.py install --user
+```
+
+This installs the package `orderly_web` for programmatic use, and a cli tool `orderly-web` for interacting with the package:
+
+## Usage
+
+```
+$ orderly-web --help
+Usage:
+  orderly_web start <path>
```

Comment:
Underscores should be dashes

Reply:
Some rogue underscores in `cli.py` also, probably worth doing a search for `orderly_web` in the repo!

Reply:
0439a5d